sql/table.go

```diff
@@ -115,6 +115,16 @@ func InsertQuery(tableName string, columnNames []string) string {
 		tableName, strings.Join(quoteColumnNames(columnNames), ","), questionMarks)
 }
 
+func InsertBulkQuery(tableName string, columnNames []string, numRecords int) string {
+	questionMarks := []string{}
+	for range numRecords {
+		questionMarks = append(questionMarks, fmt.Sprintf("(%s)", repeatComma(len(columnNames), "?")))
```

Comment:
fmt.Sprint(x) could be calculated once outside the loop

questionsmarks could be allocated with capacity of numRecords